src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -609,6 +614,7 @@ public String getShortDescription() {
     }
 
     private static final Logger LOGGER = Logger.getLogger(GitStatus.class.getName());
+    private static final int MAX_REPORTED_CONTRIBUTORS = 10;
```

Comment:
I made the limit 10 arbitrarily. Does anyone have a strong opinion on a different limit? Also, does anyone see value in allowing the limit to be set via environment variable, so that anything relying on the urls in the headers could set it as a workaround?